### PR TITLE
added a view for save-route forms and changed some css for add route …

### PR DESCRIPTION
**Pull request by Cubler**
…view

### CODE REVIEW

File: scripts/addRoute.js [hunk 1]
```diff
@@ -19,8 +19,8 @@ $(document).ready(function (){
     var larBut = document.getElementById("largeBut");
     var delBut = document.getElementById("delBut");
     var startBut = document.getElementById("startBut");
-    var saveBut = document.getElementById("saveBut");
-    var submitBut = document.getElementById("submitBut");
+    // No submitbut test*
```
Comment (rrogowski) [inline scripts/addRoute.js]: Did you mean to commit without a submit button?

Comment (rrogowski) [reply scripts/addRoute.js]: Looks like you made the save icon call `DATABASE.save()`. If we don't need the submit button code, we should remove these comments (and the ones below as well)!

Comment (Cubler) [reply scripts/addRoute.js]: That was my thought, I just wanted to run it by you before deleting it all

**Comment by rrogowski** [comment]
Looks awesome!

**Pull request merged**